source/release_notes/5.0.2-1.rst

```diff
@@ -0,0 +1,27 @@
+.. _PSMDB-5.0.2-1:
+
+================================================================================
+*Percona Server for MongoDB* 5.0.2-1
```

Comment:
Let's clearly mark it as "Release Candidate" here as well:
```suggestion
*Percona Server for MongoDB* 5.0.2-1 (Release Candidate)
```

Reply:
Added